spiped/README

```diff
@@ -21,9 +21,12 @@ Options:
 		[ipv6::addr]:port
 	Note that hostnames are resolved when spiped is launched and are not
 	re-resolved later; thus if DNS entries change spiped will continue to
-	connect to the expired address.
+	accept at the expired address.
     -t <target socket>
-	Address to which spiped should connect.
+	Address to which spiped should connect. The accepted formats are the
+	same as the ones accepted by <source socket>.
```

Comment:
`man` will word-wrap this paragraph, so I suggest removing the newline and putting "Contrary" on this same line.